### ISSUE TITLE
Cant Clone on Windows 10

### ISSUE THREAD

**Issue by Qwertin** [saleor/saleor-platform#137]
After running this command git clone https://github.com/saleor/saleor-platform.git --recursive --jobs 3

error: invalid path 'saleor/graphql/core/tests/cassettes/test_get_oembed_data[http:/www.youtube.com/watch?v=dQw4w9WgXcQ-VIDEO].yaml'
Submodule path 'saleor-dashboard': checked out '54741fbac2d973454dcceeab92b50a75e242c9f9'
Submodule path 'saleor-storefront': checked out '52ea1dd236433466b7506c286c2f82488d35d883'
fatal: Unable to checkout 'f838419945cd0d18284bcfe2689648745bbe04c5' in submodule path 'saleor'

**Comment by Qwertin** [comment]
Solved 

1. Download latest saleor relase from https://github.com/saleor/saleor/releases/
2.  Unzip to 'saleor-platform\saleor'  folder

**Comment by joshmossas** [comment]
I'm using Windows 10, and I also ran into this issue even after the PR that was supposed to fix it (https://github.com/saleor/saleor/pull/9236). 

```bash
error: invalid path 'saleor/graphql/core/tests/cassettes/test_get_oembed_data[http:/www.youtube.com/watch?v=dQw4w9WgXcQ-VIDEO].yaml'
Submodule path 'saleor-dashboard': checked out '54741fbac2d973454dcceeab92b50a75e242c9f9'
Submodule path 'saleor-storefront': checked out '52ea1dd236433466b7506c286c2f82488d35d883'
Unable to checkout 'f838419945cd0d18284bcfe2689648745bbe04c5' in submodule path 'saleor'
```

I was able to fix it by using the same method implemented by Qwertin (manually copying the saleor source code), but I think issue should probably be reopened. I don't think needing to manually copy source code is desired behavior when cloning a repo.

**Comment by NyanKiyoshi** [comment]
This is fixed on 3.1.1 (https://github.com/saleor/saleor/releases/tag/3.1.1) but this repository is still using 3.0 hence why it's not fixed here. @krzysztofwolski will be updating Saleor platform to 3.1.1
